### ISSUE TITLE
The filter do not work

### ISSUE THREAD

**Issue by Ruckhaber** [kimai/kimai#771]
I am logged in as user admin. The table timesheet is open .
I have set a filter for three users. The filter works.
Now I want to set a filter for a client. The filter does not work. I get a status bar for a long time and then
I get a message in a window with a <OK> button. The content of the message is `[object Object]`. After closing the window appears in the table timesheet: `No entries` although entries should be present.

#### Kimai version
1.0.1-19 dev

#### Relevant log entries from Kimais Debug tab
In the logfile no relevant entries are created .

**Comment by simonschaufi** [comment]
* Can you reproduce the issue on demo.kimai.org?
* there is no kimai version 1.0.1-19

**Comment by Ruckhaber** [comment]
I select 4 users and 1 costumers on demo.kimai.org. Kimai crashed and the homepage demo.kimai.org is displayed :(

**Comment by Ruckhaber** [comment]
Now it works on demo.kimai.org ??

**Comment by simonschaufi** [comment]
first create a new account on http://demo.kimai.org/status.php and then try again.

**Comment by Ruckhaber** [comment]
The filter works on demo.kimai.org.
I'm using a dev-version. The name of the downloaded file was kimai-kimai-1.0.1-19-***.zip.
The problems began with the update from version 0.9.* to version 1.0.1. I think the problem is an error in the database. Originally I use an installation on a Windows server. I have carried out a new additional installation on a Linux server and I imported the database of the other installation. I have found the same mistakes. How can I restore the version 0.9.** with the old database again .

**Comment by kevinpapst** [comment]
If you have a backup - install it. Otherwise there is no (easy) way back.
Check your database, if you have backup tables (if you did not create a backup manually).

**Comment by Ruckhaber** [comment]
I restore the database manually. In the database is located after the update a snapshot of the tables of kimai version 0.9.  I created with phpmyadmin a new empty mysql database. 
I have copied the original tables of version 0.9 into the new database. Then I renamed the tablesto original name. Next, I have installed the kimai version 0.9  and selected the new database for. 
With this trick the admin password has been reset [http://blog.rki-home.de/2014/01/22/passwort-in-kimai-zuruecksetzen/]. All user passwords have been reset to end. -> It works :)
Now kimai version 0.9 runs on Linux debian-server 3.16, PHP Version 5.6.14, Apache 2.4.10, MySQL 5.5.46
